app/pods/publications/publication/documents/template.hbs

```diff
@@ -173,6 +173,11 @@
                             {{t "document-edit"}}
                           </WebComponents::AuDropdown::Item>
                         {{/if}}
+                        {{#if (not (gt (await piece.agendaitems.length) 0))}}
+                          <WebComponents::AuDropdown::Item @href={{await piece.file.namedDownloadLink}} download={{await piece.file.downloadFilename}}>
+                            {{t "download"}}
+                          </WebComponents::AuDropdown::Item>
+                        {{/if}}
```

Comment:
piece.file. ... willen we mogelijk naar piece. ... hebben.
Dan ga je de file downloaden met de naam dat in Kaleidos te zien is.
Die kan anders zijn dan de filename (die kan je niet meer aanpassen na upload)  de piece naam wel

Vb, je kan perfect een file uploaden met text blahblah, en in de frontend aanpassen naar VR 2021 0120...
Deze download dat je hier doet gaat volgens mij dan blahblah krijgen als bestandnaam.
Zou je eens moeten testen